fortigate_connector.py

```diff
@@ -606,11 +606,18 @@ def _block_ip(self, param):
 
         # To get parameters
         try:
-            ip_addr_obj_name, policy_name, address_create_params, block_params, vdom = self._get_params(param)
+            ip_addr_obj_name, address_type, policy_name, address_create_params, block_params, vdom = self._get_params(param)
         except Exception as e:
             error_message = self._get_error_message_from_exception(e)
             return action_result.set_status(phantom.APP_SUCCESS, "Unable to create request parameters. {}".format(error_message))
 
+        # Ensure valid address type
+        if address_type not in [FORTIGATE_SRCADDR_TYPE, FORTIGATE_DSTADDR_TYPE]:
```

Comment:
Why not use `value_list` at JSON level?

Reply:
Same below

Reply:
Agreed I think a `value_list` is nicer for the user added 👍. I think this can be kept though for the app level safety to ensure no bad params were passed manually or otherwise.